utilities/virt.py

```
@@ -1538,17 +1530,14 @@ def service_port(self):
 def wait_for_ssh_connectivity(
     vm: VirtualMachineForTests, timeout: int = TIMEOUT_2MIN, tcp_timeout: int = TIMEOUT_1MIN
```

Ruff (0.14.13)

Unused function argument: `tcp_timeout`

(ARG001)